src/sim.rs

```diff
@@ -34,7 +34,7 @@ pub struct Sim<'a> {
 }
 
 impl<'a> Sim<'a> {
-    pub(crate) fn new(config: Config, world: World) -> Self {
+    pub fn new(config: Config, world: World) -> Self {
```

Comment:
I don't believe this one should change. Construction should happen through the builder.